From 0d40091e25ab69e9f29eaa13269847e9db55db12 Mon Sep 17 00:00:00 2001
From: rajeshvenkata <rajesh.venkat.p@gmail.com>
Date: Wed, 28 Jan 2026 00:00:00 +0000
Subject: [PATCH 1/2] Disable secondstar and UI services

EKS-A does not use the secondstar (SSH over serial) service or the
web UI. Disable both by default in the helm chart values to reduce the
consolidated tinkerbell deployment's memory footprint and attack surface.

Signed-off-by: rajeshvenkata <rajesh.venkat.p@gmail.com>
---
 helm/tinkerbell/values.yaml | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/helm/tinkerbell/values.yaml b/helm/tinkerbell/values.yaml
index 8fe09c83..9f0215db 100644
--- a/helm/tinkerbell/values.yaml
+++ b/helm/tinkerbell/values.yaml
@@ -32,12 +32,12 @@ deployment:
       dualStack: false
       enableCRDMigrations: true
       enableRufioController: true
-      enableSecondstar: true
+      enableSecondstar: false
       enableSmee: true
       enableTootles: true
       enableTinkServer: true
       enableTinkController: true
-      enableUI: true
+      enableUI: false
       httpPort: 7080
       httpsPort: 7443
       logLevel: 0
-- 
2.46.0

